docs/metrics.md

```diff
@@ -15,7 +15,7 @@ Resulting in a configuration like the following:
 ```
 thresholdMode:
   threshold: 2
-  cosigners:
+  cosigner:
```

Comment:
guessing this was accidentally changed, same for the other occurrence on L100

Reply:
Yeah.